File: v1/src/main/java/com/api/v1/dtos/rent/RentResponseDto.java
package com.api.v1.dtos.rent;

import com.api.v1.dtos.payment.PaymentResponseDto;
import com.api.v1.dtos.car.CarResponseDto;
import com.api.v1.dtos.customer.CustomerResponseDto;

import java.math.BigDecimal;
import java.math.BigInteger;

public record RentResponseDto(
        BigInteger orderNumber,
        CustomerResponseDto customer,
        CarResponseDto car,
        PaymentResponseDto payment,
        int days,
        BigDecimal price,
        String returningDate,
        String returnedAt,
        String rentedAt
) {
}

File: v2/src/main/kotlin/com/api/v2/rent/dtos/RentResponseDto.kt
package com.api.v2.rent.dtos

import com.api.v2.car.dtos.CarResponseDto
import com.api.v2.customer.dtos.CustomerResponseDto
import com.api.v2.payment.dtos.PaymentResponseDto
import java.math.BigInteger

data class RentResponseDto(
    val orderNumber: BigInteger,
    val customer: CustomerResponseDto,
    val car: CarResponseDto,
    val payment: PaymentResponseDto,
    val rentedAt: String
)
